docs/geometry.rst

```diff
@@ -71,6 +71,41 @@ which this geometry code can position independently.
 
    .. automethod:: from_h5_file_and_quad_positions
 
+   .. automethod:: to_distortion_array
+
+   .. automethod:: plot_data_fast
+
+   .. automethod:: position_modules_fast
+
+   .. automethod:: inspect
+
+DSSC-1M
+-------
+
+DSSC-1M consists of 16 modules of 128×512 pixels each.
+Each module is further subdivided into 2 sensor tiles,
+which this geometry code can position independently.
+
+.. figure:: _static/dssc_layout.png
+
+   The approximate layout of DSSC-1M, in a front view (looking along the beam).
+
+The pixels in each DSSC module are tesselating hexagons.
```

Comment:
tessellating: After living for more than 5 years in an English speaking country I am still learning vocabulary. I am sure I will have forgotten this word by tomorrow.  I think you could use the word _mosaic_, which I believe is more universal in other languages and makes it more unlikely that people like me stumble on it.

Reply:
Maybe we can just leave out the word altogether? I don't think mosaic necessarily means tessellating - lots of mosaics have gaps between the tiles to make the desired shapes.

Reply:
Nah, that's fine. If you want to keep it just keep it